package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "ascii-table": "0.0.9",
     "fast-equals": "^4.0.3",
+    "localforage-removeitems": "^1.4.0",
```

Comment:
I think this should be a `peerDependency` instead of a `dependency`

Reply:
Fixed, thanks